### PR TITLE
Fix terms page

### PR DESCRIPTION
**Pull request by xbpcb**
Use the same styles for both Terms and Documentation pages. Fix Terms page,

### CODE REVIEW

File: src/views/pages/terms.html [hunk 1]
```diff
@@ -11,7 +11,11 @@
 	<c-header></c-header>
 
 	<div class="p-privacy-policy">
-		<div class="dropdown">
+		<div class="please-wait-sign {{#if @shared.auNetText && @shared.ppNetText && @shared.ppComText}} not-displayed{{/if}}">
```
Comment (MartinKolarik) [inline src/views/pages/terms.html:14]: I have to say I don't understand this logic at all - we're showing the loading message if all those variables **exist**? It seems to actually work but the code seems like it should do the exact opposite.

**Review by MartinKolarik — COMMENTED** [review]
Not sure why but the main navbar is hidden until the docs load, which it shouldn't.

File: src/views/pages/terms.html (hunk 1 above)
Comment (xbpcb) [reply src/views/pages/terms.html:14]: Nope, we adding the class .not-displayed which will hide the message once we got all data we need

Comment (xbpcb) [reply src/views/pages/terms.html:14]: > Not sure why but the main navbar is hidden until the docs load, which it shouldn't.

I will check it

Comment (jimaek) [reply src/views/pages/terms.html:14]: Any news on this?

Comment (xbpcb) [reply src/views/pages/terms.html:14]: I can't reproduce this bug no more. I've checked the code and find nothing, testing with throttling, different browsers etc. - nothing.

**Pull request merged**